# Generated by Django 3.1.1 on 2021-01-26 06:00

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
from django.contrib.auth.models import User

def forwards_func(apps, schema_editor):
    User.objects.create(username="admin",password="admin123432",email="admin@gmail.com")

class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     
        

    ]

    operations = [
  
        migrations.CreateModel(
            name='Profile',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('city', models.CharField(blank=True, max_length=360)),
                ('area', models.CharField(blank=True, max_length=360)),
                ('phone', models.CharField(blank=True, max_length=16)),
                ('email_confirmed', models.BooleanField(default=False)),
                ('ps', models.CharField(blank=True, max_length=499, null=True)),
                ('gender', models.CharField(blank=True, choices=[('m', 'Male'), ('f', 'Female'), ('o', 'Other')], max_length=1)),
                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]





# def add_user(apps, schema_editor): 
#     apps.get_model('myapp', 'Foo').objects.filter(user_id=1).update(
#         user=apps.get_model('auth', 'User').objects.create_user(username='anonymous')
#     )